code.js

```diff
@@ -0,0 +1,115 @@
+const cost = 'Cur_OfficialRate';
```

Comment:
Global constants need to write in UPPER_CASE style

Reply:
done